### PR TITLE
Edited Remote Domains section

### PR DESCRIPTION
**Pull request by Ajjos**
Have edited & added the Remote domain scenarios with their exact corresponding settings in Outbound Spam policy as current information provided lacks clarity and is causing confusion.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 kindly do the copy editor review for this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder.

**Comment by chrisda** [comment]
@Ajjos, I made some updates/clarifications to your updates. Please take a look and make any updates as you see fit.

I believe that **Automatic - System controlled** now means **Off** for everyone, based on the information here:

<https://techcommunity.microsoft.com/t5/exchange-team-blog/all-you-need-to-know-about-automatic-email-forwarding-in/ba-p/2074888>

Per that blog post, it used to mean **On**, but it was changed everywhere for everyone to mean **Off**. I'm willing to admit that the change might not be globally complete, but it's definitely heading that direction.

File: microsoft-365/security/office-365-security/external-email-forwarding.md [hunk 1]
```diff
@@ -59,12 +59,20 @@ As an admin, you might have already configured other controls to allow or block
...
-- You allow automatic forwarding for a remote domain, but you block automatic forwarding in outbound spam filter policies. In this example, automatically forwarded messages are blocked.
-- You allow automatic forwarding in outbound spam filter policies, but you use mail flow rules or remote domain settings to block automatically forwarded email. In this example, the mail flow rules or remote domain settings will block automatically forwarded messages.
+<br>
 
-This feature independence allows you to (for example) allow automatic forwarding in outbound spam filter policies, but use remote domains to control the external domains that users can forward messages to.
+****
+
+|Scenario|Result|
+|---|---|
+|<ul><li>You configure remote domain settings to allow automatic forwarding.</li><li>Automatic forwarding in the outbound spam filter policy is set to **Off**.</li></ul>|Automatically forwarded messages to recipients in the affected domains are blocked.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/external-email-forwarding.md:70]:
```suggestion
|<ul><li>You configure remote domain settings to allow automatic forwarding.</li><li>Automatic forwarding in the outbound spam filter policy is set to **Off**.</li></ul>|Automatically forwarded messages to recipients in the affected domains are blocked.|
```

File: microsoft-365/security/office-365-security/external-email-forwarding.md [hunk 2]
```diff
@@ -59,12 +59,20 @@ As an admin, you might have already configured other controls to allow or block
...
+|<ul><li>You configure remote domain settings to allow automatic forwarding.</li><li>Automatic forwarding in the outbound spam filter policy is set to **Automatic - System-controlled**.</li></ul>|Automatically forwarded messages to recipients in the affected domains are blocked. <p> As described earlier, **Automatic - System-controlled** used to mean **On**, but the setting has changed over time to mean **Off** in all organizations. <p> For absolute clarity, you should configure your outbound spam filter policy to **On** or **Off**.|
+|<ul><li>Automatic forwarding in the outbound spam filter policy is set to **On**</li><li>You use mail flow rules or remote domains to block automatically forwarded email.</li></ul>|Automatically forwarded messages to affected recipients are blocked by mail flow rules or remote domains.|
+|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/external-email-forwarding.md:73]:
```suggestion
```

**Comment by Ajjos** [comment]
The commits are perfect now, let's publish the changes into the public Docs article.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by Ajjos** [comment]
Great, thank you so much guys. Looking forward for more commits & PRs in
future.


Best Regards

Ajay Joshi
8595351661

**Comment by Ajjos** [comment]
Hi all,

I checked the public article and don't find the changes updated as yet.
When can we expect this to happen?

https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/external-email-forwarding?view=o365-worldwide


Best Regards

Ajay Joshi
8595351661

**Comment by tiburd** [comment]
@Ajjos 
Hi, Ajay,

It looks like the changes are now included in the article.

In most cases, publishing to live happens around 10:30 AM and 3:30 PM Pacific Time, Monday through Friday, except on holidays. It may take 15-30 minutes to publish all the repos and for changes to appear on the live site.

Thanks!

**Comment by Ajjos** [comment]
Hi Tina,

Thanks for the update but I still don't see the changes. Could you please
share the link?

This is the article link I'm referring to and also attached the screenshot
where we intended the changes will be made(bullet points will be expanded &
3rd point inserted -
https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/external-email-forwarding?view=o365-worldwide
.



Best Regards

Ajay Joshi
8595351661

**Comment by Ajjos** [comment]
Hi all,

Thanks for everyone's contribution, the changes have been made live now.


Best Regards

Ajay Joshi
8595351661

**Pull request merged**